### ISSUE TITLE
Where is the ‘clip_trainable_vars‘ used?

### ISSUE THREAD

**Issue by CenGuandong** [duxingren14/DualGAN#23]
Where is the ‘clip_trainable_vars‘ used?

**Comment by Hx96** [comment]
i have the same question, i found that a issue  point out that the author change to the traditional loss.so dont use w gan clip

**Comment by duxingren14** [comment]
Yes. I have updated the codes a few times and disabled wgan loss Sorry for causing the confusion. In the near future, I will provide both options (wgan and nsgan) for convenience to reproduce the paper results.
